xontrib-cookiecutter v2 (#2)
xonsh/xontrib-cookiecutter: xontrib-{{cookiecutter.project_slug}}/README.md

<p align="center">
{{cookiecutter.project_short_description}}
</p>

<p align="center">  
If you like the idea click ⭐ on the repo and stay tuned.
</p>

## Installation

To install use pip:

```bash
xpip install xontrib-{{cookiecutter.project_slug}}
# or: xpip install -U git+https://github.com/{{cookiecutter.github_username}}/xontrib-{{cookiecutter.project_slug}}
```

## Usage

```bash
xontrib load {{cookiecutter.project_slug}}
# TODO: what's next?
```

## Examples

...

## Known issues

...

## Credits

This package was created with [xontrib cookiecutter template](https://github.com/xonsh/xontrib-cookiecutter).

--------------------

## xontrib promotion (READ and REMOVE THIS SECTION)

> After you create the xontrib repository you can do some helpful tasks to spread the words about xontrib.

Reviewer: ```suggestion
After you create the xontrib repository you can do some helpful tasks to spread the word about your xontrib.
```

> **Repository name**. It's a good practice to add `xontrib-` prefix before the name of your repository. It allows use the Github search to find it.

Reviewer: ```suggestion
**Repository name**. It's a good practice to add `xontrib-` prefix before the name of your repository. It helps Github search find it.
```

> **Add topics to the repository**. To show the xontrib repository in Github Topics please add topics `xonsh` and `xontrib` to the repository "About" setting. Also add thematic topics for example `ssh` if your xontrib improves of working with it.

Reviewer: ```suggestion
**Add topics to the repository**. To show the xontrib repository in Github Topics please add topics `xonsh` and `xontrib` to the repository "About" setting. Also add thematic topics, for example,  `ssh` if your xontrib helps work with `ssh`.
```
